internal/v2/command/user.go

```diff
@@ -138,6 +138,14 @@ func (r *CommandSide) RemoveUser(ctx context.Context, userID, resourceOwner stri
 	return r.eventstore.PushAggregate(ctx, existingUser, userAgg)
 }
 
+func (r *CommandSide) isUserActive(ctx context.Context, userID, resourceOwner string) (bool, error) {
```

Comment:
maybe isUserExisting would be better. Because when the user is deactivated this will return true

Reply:
now called checkUserExists